app/workers/kms_encryption_verification_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class KmsEncryptionVerificationJob
+  include Sidekiq::Worker
+
+  def perform(models = ApplicationRecord.descendants_using_encryption.map(&:name))
+    models.map(&:constantize).each do |model|
+      keys = model.lockbox_attributes.keys
```

Comment:
```suggestion
      encrypted_attributes = model.lockbox_attributes.keys
```

These aren't the encryption keys, right? These are the db fields/model attributes?

Reply:
Yeah they're the model attributes